lib/code_climate/test_reporter/git.rb

```diff
@@ -34,9 +38,13 @@ def head
           git("log -1 --pretty=format:'%H'")
         end
 
-        def committed_at
+        def committed_at_from_ci
+          Ci.service_data[:committed_at]
+        end
+
+        def committed_at_from_git
           committed_at = git("log -1 --pretty=format:%ct")
-          committed_at.to_i.zero? ? nil : committed_at.to_i
+          committed_at.nil? ? nil : committed_at.to_i
```

Comment:
I believe `#git` will return an empty string, not nil in this case. If that's right, this should remain unchanged